bin/analyze.sh

```diff
@@ -1,6 +1,6 @@
-#!/bin/bash
+#!/bin/sh
 
 # Usage:
 # ./bin/analyze.sh two_fer ~/test/
-
-python bin/analyze.py $1 $2
\ No newline at end of file
+export PYTHONPATH=/opt/analyzer/lib/$1/
```

Comment:
Isn't the analyze.py doing this already? Perhaps it's doing its job wrong and we should fix it there or remove that piece completely?